app/models/funding/trainee_summary_row.rb

```diff
@@ -55,5 +57,16 @@ def route
 
       self.class.human_attribute_name(training_route)
     end
+
+    def sync_lead_school_and_partner_name
+      school_name_changed = changed.include?("lead_school_name")
+      partner_name_changed = changed.include?("lead_partner_name")
+
+      if school_name_changed
+        self.lead_partner_name = lead_school_name
```

Comment:
Could we regenerate the `== Schema Information` section in this file to include the new field?